Assets/Scripts/Game/EnemySenses.cs

```diff
@@ -116,6 +144,52 @@ void FixedUpdate()
             else
                 classicUpdate = false;
 
+            // Update whether enemy would be spawned or not in classic.
+            // Only check if within the maximum possible distance (Just under 1094 classic units)
+            if (classicUpdate && distanceToPlayer < 1094 * MeshReader.GlobalScale)
```

Comment:
Just realized, that if the player teleported from being close to an enemy to outside this distance, the enemy's "spawned in classic" state would be stuck at whatever it was.